api/tests/integration/infrastructure/repositories/campaign-participation-repository_test.js

```diff
@@ -10,6 +10,73 @@ const DomainTransaction = require('../../../../lib/infrastructure/DomainTransact
 const { STARTED, SHARED, TO_SHARE } = CampaignParticipationStatuses;
 
 describe('Integration | Repository | Campaign Participation', function () {
+  describe('#getCodeOfLastParticipationToProfilesCollectionCampaignForUser', function () {
+    let userId;
+    const expectedCode = 'GOOD';
+    afterEach(async function () {
+      await knex('campaign-participations').delete();
+    });
+    beforeEach(async function () {
+      userId = databaseBuilder.factory.buildUser().id;
+      await databaseBuilder.commit();
+    });
+
+    it('should return null if there is no participations', async function () {
+      // when
+      const code = await campaignParticipationRepository.getCodeOfLastParticipationToProfilesCollectionCampaignForUser(
+        userId
+      );
+
+      // then
+      expect(code).to.equal(null);
+    });
+
+    it('should return null if there is no participations to share', async function () {
```

Comment:
**suggestion** : rajouter un test 'should return null if there is no profiles collection campaign'